frame/vault/src/lib.rs

```diff
@@ -487,6 +490,26 @@ pub mod pallet {
 			Self::deposit_event(Event::VaultStarted { vault });
 			Ok(().into())
 		}
+		#[pallet::weight(10_000)]
+		pub fn liquidate_strategy(
```

Comment:
`liquidate` is a bit misleading IMO, looks like the caller is trying to pay someone's debt

Reply:
You can rename it if you feel like it is misleading :)

Reply:
On a second thought, I think the name should remain as is